docs/tutorials/getting_started/usage.old

```diff
@@ -479,7 +479,7 @@ Here is a with an even larger initial number of :code:`Defector` (note that it t
     eco = axelrod.Ecosystem(results, population=[.1, .05, 7, .1, .05])
     eco.reproduce(140) # Evolve the population over 140 time steps
     plot = axelrod.Plot(results)
-    p = plot.stackplot(eco.population_sizes)
+    p = plot.stackplot(eco)
```

Comment:
This doesn't matter terribly as really all the `.old` files should be deleted at some point *but* still good to not have any legacy code lying around.